src/ui_framework/doc_site/src/components/guide_code_viewer/guide_code_viewer.jsx

```diff
@@ -0,0 +1,76 @@
+
+import React, {
+  Component,
+  PropTypes,
+} from 'react';
+
+import classNames from 'classnames';
+import hljs from 'highlight.js';
+
+export default class GuideCodeViewer extends Component {
+
+  constructor(props) {
```

Comment:
Same comment here... though I'm thinking once we add the linting rule for this, we can get rid of them all at once.